theseus/utilities/loggers/wandb_logger.py

```diff
@@ -1,8 +1,5 @@
-try:
-    import theseus.utilities.loggers.wandb_logger as wandb_logger
-except ModuleNotFoundError:
-    pass
 import torch
+import wandb as wandb_logger
```

Comment:
same as above, I don't want to obligate the user to install `wandb` if they don't use it in the project

Reply:
actually, I think there is a small mistake here that I've made. Can you please put this line inside try ... except